dev-qt/qtwebengine/qtwebengine-5.15.0.ebuild

```diff
@@ -85,6 +89,11 @@ PATCHES=(
 )
 
 src_prepare() {
+	if use ppc64; then
+		# ppc64 patches may lag behind for couple of days, do not block other arches
+		# ping gyakovlev@gentoo.org if you have any problems on ppc64
+		eapply "${WORKDIR}/${PF}-ppc64"
```

Comment:
If you expect the patchset to change rarely, using `PF` here doesn't seem right

Reply:
true, I have version in  the filenames and will just name the directory inside tarball as `qtwebengine-ppc64`